tests/runtest.py

```diff
@@ -449,17 +449,17 @@ def mermaid_sort(self, output, ignore_children=False):
         start_mermaid = False
 
         for ln in output.split('\n'):
-            if ln.find('<div class=\"mermaid\">') >= 0:
+            if ln.find('<div class=\"mermaid\"') >= 0:
                 start_mermaid = True
                 continue
             if start_mermaid == False:
                 continue
             if ln.find('</div>') >= 0:
                 break
 
-            m = re.match( r'\s+(?P<start_id>\d+)\[\"(?P<start_name>\S+)\"\]\s+-->\|(?P<call_num>\d+)\|\s+(?P<end_id>\d+)\[\"(?P<end_name>\S+)\"\];', ln)
+            m = re.match( r'\s+(?P<start_depth>\d+)_(?P<start_id>\d+)\[\"(?P<start_name>\S+)\"\]\s+-->\|(?P<call_num>\d+)\|\s+(?P<end_depth>\d+)_(?P<end_id>\d+)\[\"(?P<end_name>\S+)\"\];', ln)
```

Comment:
Could you please this line shorter?  You can break it in multiple lines if possible.